src/Internal.AspNetCore.Sdk/build/Common.targets

```diff
@@ -15,6 +15,16 @@ For single-tfm projects, this will be imported from build/Internal.AspNetCore.Sd
     <IncludeSymbols Condition="'$(NuspecFile)'!=''">false</IncludeSymbols>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(IncludeSymbols)' != 'false' ">
+    <!-- Include .pdb files in regular .nupkg files. -->
+    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
+  </PropertyGroup>
+
+  <Target Name="_EnsurePdbsInNupkgArePortable" BeforeTargets="GenerateNuspec" Condition=" '$(IsPackable)' != 'false' ">
+    <Error Text="Symbols included in a nupkg package should be portable. Set DebugType=portable or IncludeSymbols=false."
+      Condition=" '$(DebugType)' != 'portable' AND '$(IncludeSymbols)' == 'true' " />
```

Comment:
Btw, the last I looked, symbols for Razor's net46 binaries are `full`. Can you verify this works for the repo?

Reply:
Why are they `full`? Don't we need `portable` for these to work for x-plat?

Reply:
I'm not entirely sure. @NTaylorMullen might know more.

Reply:
Looks like it's due to this, right? https://github.com/aspnet/Razor/issues/1548 If we need `full` for VS, let's still produce `portable` by default. We can convert `portable` symbols to `full` in our VS publishing steps. I believe @muratg knows how to do that.

Reply:
Ahh, yeah. Completely forgot I made that change 👍